tests/update_skll_models.py

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env python
 
 """
-This script is designed to update the sci-kit learn model files to ensure they
-are compatible with the current version.
-The script goes through all tests in the data/experiments directory, finds the model files and
-saves them again.
-Note that this will overwrite the original test data.
-Before running this script, make sure that the outputs work as expected
+Update SKLL models in tests.
+
+This script updates the SKLL model files to ensure they are compatible with the
+current version. It simply through all the rsmprednt tests under the
```

Comment:
```suggestion
current version. It simply iterates through all the rsmprednt tests under the
```

Reply:
Is `rsmprednt` a typo too? (Sorry, wouldn't have noticed this if not for @aloukina's suggestion.)

Reply:
Yep, already fixed 👍